app/helpers/application_helper.rb

```diff
@@ -1,3 +1,11 @@
 module ApplicationHelper
   # Returns the full title on a per-page basis.
+  def full_title page_title = ""
+    base_title = t("dictionary.main_title")
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
```

Comment:
1. đoạn if - else này em dùng toán tử ngôi số 3 đi
`condition ? result_for_true : result_for_false`

2. dùng blank? thay vì empty? nhé
cho chị bjk sự khác nhau luôn